.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "ced"]
+	path = ced
+	url = https://Lord-Kamina@github.com/performous/compact_enc_det
```

Comment:
make this link universal please, when I wanted to commit a fix it started to ask for your password.

Reply:
Hahahaha hadn't even noticed.